compensated-spec/features/steps.rb

```diff
@@ -1,13 +1,26 @@
+ParameterType(
+  name: 'spec_helper',
+  regexp: /`.*`/,
+  transformer: ->(s) { s.gsub("`", '') },
+  use_for_snippets: true
+)
```

Comment:
I read this: https://cucumber.io/docs/cucumber/cucumber-expressions/#custom-parameter-types but I still don't know what does `use_for_snippets: true` do.
Mind giving me an example?

Reply:
`use_for_snippets` ensures that when Cucumber cannot find a step definition, but it thinks a custom parameter type matches a particular chunk of the string, it will include the parameter type in the example step it shows the tester.

I _believe_ `use_for_snippets` is true by default, but I was copy-pasting from an example that had `use_for_snippets` set to false.